readme.md

```diff
@@ -116,6 +117,11 @@ Open DevTools for the specified `BrowserWindow` instance or the focused one.
 Type: `BrowserWindow`\
 Default: The focused `BrowserWindow`
 
+### preloadScriptPath
+
+The absolute path to a preload script that will enable devtron, when node integration is off.
```

Comment:
This sentence is not correct. What it says is that the absolute path only exists when the node integration is off. Please put some more effort into this.

`node` => `Node.js`

`node integration` => should be linkified to some docs about the `nodeIntegration` option.

Reply:
Sounded fine to me as a non-native speaker, sorry. Hope it's better now.